test/MigrationsConfigurationFactoryTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare(strict_types=1);
+
+namespace RoaveTest\PsrContainerDoctrine;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Schema\AbstractSchemaManager;
+use PHPUnit\Framework\TestCase;
+use Psr\Container\ContainerInterface;
+use Roave\PsrContainerDoctrine\AbstractFactory;
+use Roave\PsrContainerDoctrine\MigrationsConfigurationFactory;
+
+final class MigrationsConfigurationFactoryTest extends TestCase
+{
+    public function testExtendsAbstractFactory() : void
+    {
+        $this->assertInstanceOf(AbstractFactory::class, new MigrationsConfigurationFactory());
+    }
+
+    public function testConfigValues() : void
+    {
+        $connection = $this->buildConnection();
+        $container  = $this->createMock(ContainerInterface::class);
+
+        $config = [
+            'doctrine' => [
+                'migrations_configuration' => [
+                    'orm_default' => [
+                        'directory' => 'test/TestAsset',
+                        'name'      => 'Foo Bar',
+                        'namespace' => 'Acme\Lib\Migrations',
+                        'table'     => 'baz',
+                        'column'    => 'ver',
+                    ],
+                ],
+            ],
+        ];
+
+        $container->expects($this->exactly(2))
```

Comment:
Similar to below. Also better: `returnValueMap` instead of consecutive calls, since order isn't important